client/components/grid-column.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="grid-column" :style="{ maxWidth: width }">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'grid-column',
+  props: ['width'],
+};
+</script>
+
+<style lang="stylus">
+.grid-column {
+  flex-grow: 1;
+  margin-right: 5px;
```

Comment:
Can you use grid-gap instead of margin-right?

Reply:
good catch. i'll update.

Reply:
looks like this doesn't work on display: flex